CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+cmake_minimum_required(VERSION 3.12)
+project(aerosol-radiation)
+enable_language(Fortran)
+
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
This is pretty aggressive! Are there working C++17 compilers now? :-)

Reply:
Maybe 11? I copied some of this from another small repo I was trying out some things that used C++17

Reply:
11 or 14 should be okay. Let's start with 11 and see what's "missing."